tests/bluetooth/tester/src/btp_bap.c

```diff
@@ -44,7 +45,10 @@ static const struct bt_audio_codec_cap vendor_codec_cap = BT_AUDIO_CODEC_CAP(
 
 struct audio_stream {
 	struct bt_bap_stream stream;
-	uint8_t ase_id;
+	union {
+		uint8_t ase_id;
+		uint8_t bis_id;
+	};
```

Comment:
It would make more sense to have a union between the `bis_id` and the `cis_id`. 

It's not a bad idea to split the data by unicast/broadcast, but in that case I would do a proper split rather than just this single field.